js/scripts.js

```diff
@@ -1,3 +1,14 @@
-var name = prompt('Enter your name');
-alert('Hello, ' + name);
-console.log('Hello, ' + name);
+// var name = prompt('Enter your name');
+// alert('Hello, ' + name);
+// console.log('Hello, ' + name);
+
+var a = prompt('Type a');
+var h = prompt('Type h');
+function getTriangleArea(a, h) {
+if (a, h) <= 0) {
+return ('Nieprawidłowe dane');
+} if else (a, h) > 0) {
```

Comment:
Ne ma czegoś jak `if else`. W ogóle ten warunek jest niepotrzebny bo jeśli dane będą nieprawidłowe to funkcja ucieknie na returnie z 9 linijki. Więc skoro funkcja doszła do 10 linijki i nie uciekła to znaczy, że dane są ok.

Warunek oprócz tego, że niepoprawny to i niepotrzebny.